win32ss/user/winsrv/consrv/condrv/text.c

```diff
@@ -977,6 +977,18 @@ IntWriteConsoleOutputStringUnicode(
         if (bCJK && Ptr->Char.UnicodeChar >= 0x80 &&
             mk_wcwidth_cjk(Ptr->Char.UnicodeChar) == 2)
         {
+            if (X == Buffer->ScreenBufferSize.X - 1)
```

Comment:
Please add a comment saying that we go to the next line because a double-width character cannot cross a line boundary.

Reply:
OK.